src/LinkDotNet.Blog.Web/RegistrationExtensions/StorageProviderExtensions.cs

```diff
@@ -49,6 +51,32 @@ public static void AddStorageProvider(this IServiceCollection services, IConfigu
         }
     }
 
+    public static void InitializeStorageProvider(this WebApplication app, IConfiguration configuration)
```

Comment:
Ideally, this would be registered with the other StorageProvider stuff in their respective extension method, eliminating the need for the SQL checks.
But then again: See my last comment - I am leaning away from the automatic migration as we don't have many migrations.